spec/griddler/cloudmailin/adapter_spec.rb

```diff
@@ -47,4 +124,25 @@ def default_params
       EOS
     }
   end
+
+  def bcc_params
+    default_params.merge(
+      envelope: default_params[:envelope].merge(to: 'sandra@example.com')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.